tests/testthat/_snaps/profile_emissions.md

```diff
@@ -25,18 +25,21 @@
        [1] "companies_id"                       "company_name"                      
        [3] "country"                            "emission_profile_share"            
        [5] "emission_profile"                   "benchmark"                         
-       [7] "co2e_lower"                         "co2e_upper"                        
-       [9] "matching_certainty_company_average" "company_city"                      
-      [11] "postcode"                           "address"                           
-      [13] "main_activity"                      "profile_ranking_avg"               
+       [7] "matching_certainty_company_average" "company_city"                      
+       [9] "postcode"                           "address"                           
+      [11] "main_activity"                      "profile_ranking_avg"               
+      [13] "co2_avg"                            "isic_4digit"                       
+      [15] "tilt_sector"                        "tilt_subsector"                    
+      [17] "unit"                               "co2e_lower"                        
+      [19] "co2e_upper"                        
 
 # informs the mean noise percent
 
     Code
       invisible <- profile_emissions(companies, co2, europages_companies,
         ecoinvent_activities, ecoinvent_europages, isic_name)
     Message
-      i Adding 48% and 99% noise to `co2e_lower` and `co2e_upper`, respectively.
+      i Adding 49% and 103% noise to `co2e_lower` and `co2e_upper`, respectively.
 
```

Comment:
The algorithm changed for how we summarize `co2_footprint` so it makes sense that the numbers are now different and therefore the noise is also different.